CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** Ask in [our Slack channel]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Issue]
+- **💻 I want to code:** See below
+
+If you like the project, but just don't have time to contribute, that's OK too!
+You can also star the project ⭐, rave about it online 💬, or add a link to us
+🔗 in your project's readme.
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead, please [report them to the Microsoft Security Response Center]. You can
+read more about our security policy in [`SECURITY.md`].
+
+## Contributing code
+
+👨‍💼 This repository is a collection of dev container Features managed by dev
+container spec maintainers. Even though **we do not accept new Features in this
+repository**, we welcome contributions to [our existing Features]!
+
+🚀 If you're looking to create a new Feature, we encourage you to [create your
+own Feature repository] and add it to the index in [`collection-index.yml`].
+
+👷‍ If you've identified an issue and you want to fix it, here's how you can get
+started:
+
+1. 🔀 Fork the repo
+2. 💻 Open the repo in your editor
+3. 👨‍💻 Add your changes to your workspace
+4. ✨ Run the tests to make sure everything works
```

Comment:
Can we add a reference link to https://github.com/devcontainers/cli/blob/main/docs/features/test.md? thanks!

Reply:
Fixed in https://github.com/jcbhmr/features/commit/3aa8264f0cabb849ba3487028a27803a67cac91d